adapters/from_sql.py

```diff
@@ -0,0 +1,187 @@
+"""Converts the data from SQL to expected format."""
+import json
+from data.release import Release
+from data.task import Task
+from adapters.to_sql import to_sql_tasks
+from adapters.to_sql import to_sql_xcom
+from resources.to_timestamp import to_timestamp
+
+
+def read_releases(release_data, airflow_db):
```

Comment:
why don't you move all the methods that takes in an airflow db to the AirflowDB class ?

Reply:
can we do a join select instead ? would it be possible to get all the information in one query ?

Reply:
Sure, I can move them into the AirflowDB class, though I feel as if it might make more sense to have those that just take in airflow_db just so that it can pass it off to other functions in the AirflowAdapter class?

Reply:
I'm not too well-versed in SQL, but it does look like it might be possible to get all of the info in one query. It would also push more of the data retrieval onto the SQL server, so I'll go ahead and implement it.

Reply:
Maybe that s a little bit too soon to do that, but eventually what I want to get is getting the raw data from Airflow is one thing, and converting the data is another. Let's do that in a future PR once we have all the required data.